### PR TITLE
[Improve] Harmless changes according to Pylint

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: mmaction/apis/test.py [hunk 1]
```diff
@@ -181,3 +180,4 @@ def collect_results_gpu(result_part, size):
         # the dataloader may pad some samples
         ordered_results = ordered_results[:size]
         return ordered_results
+    return None
```
Comment (innerlee) [inline mmaction/apis/test.py:183]: double check that None is a valid return value

Comment (dreamerlin) [reply mmaction/apis/test.py:183]: `None` is a valid return value by checking

File: mmaction/datasets/pipelines/augmentations.py [hunk 2]
```diff
@@ -252,10 +252,7 @@ def __init__(self, flip_ratio=0.5, direction='horizontal'):
         self.direction = direction
 
     def __call__(self, results):
-        if np.random.rand() < self.flip_ratio:
-            flip = True
-        else:
-            flip = False
+        flip = bool(np.random.rand() < self.flip_ratio)
```
Comment (innerlee) [inline mmaction/datasets/pipelines/augmentations.py:255]: ~~bool~~

File: mmaction/models/backbones/x3d.py [hunk 3]
```diff
@@ -391,7 +393,7 @@ def make_res_layer(self,
         if self.se_style == 'all':
             use_se = [True] * blocks
         elif self.se_style == 'half':
-            use_se = [True if i % 2 == 0 else False for i in range(blocks)]
+            use_se = [bool(i % 2 == 0) for i in range(blocks)]
```
Comment (innerlee) [inline mmaction/models/backbones/x3d.py:396]: ~~bool~~

**Comment by innerlee** [comment]
generally okay

**Review by innerlee — APPROVED** [review]

**Pull request merged**